src/dotnet-new3/Program.cs

```diff
@@ -23,6 +23,11 @@ public static class Program
 
         public static int Main(string[] args)
         {
+            if (Console.IsOutputRedirected)
+            {
+                Console.OutputEncoding = System.Text.Encoding.UTF8;
```

Comment:
I guess it's a bit more than just removal :P

Reply:
have to added it because the tests were failing without it